test/exports.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const { strictEqual } = require('assert');
+const { valid } = require('semver');
+
+const nodeAddonApi = require('../');
+
+module.exports = function test () {
+  strictEqual(nodeAddonApi.include.startsWith('"'), true);
+  strictEqual(nodeAddonApi.include.endsWith('"'), true);
+  strictEqual(nodeAddonApi.include.includes('node-addon-api'), true);
+  strictEqual(nodeAddonApi.include_dir, '');
+  strictEqual(nodeAddonApi.gyp, 'node_api.gyp:nothing');
+  strictEqual(nodeAddonApi.targets, 'node_addon_api.gyp');
+  strictEqual(valid(nodeAddonApi.version), true);
```

Comment:
It may be worth checking the version explicitly here. Can we compare it to the package.json version?

Reply:
Thanks for the review. Would something like the following be suitable?
```suggestion
  strictEqual(valid(nodeAddonApi.version), true);
  strictEqual(nodeAddonApi.version, require('../package.json').version);
```

Reply:
+1 on the change.

Reply:
Thanks both, I've added this test expectation.